nginx/nginx.conf.template

```diff
@@ -1,10 +1,14 @@
+## ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ##
+## ! This file was generated from a template ! ##
+## ! See /etc/nginx/templates/               ! ##
+## ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ! ##
 daemon off;
 user www-data;
-worker_processes 4;
+worker_processes ${NGINX_WORKERS};
```

Comment:
Now that I'm thinking about it, maybe the variables should be:

- `NGINX_WORKER_PROCESSES` => `worker_processes`
- `NGINX_WORKER_CONNECTIONS` => `worker_connections`

?

Reply:
Fixed